src/packet/signature.js

```diff
@@ -509,7 +538,7 @@ class SignaturePacket {
         // Issuer Fingerprint
         this.issuerKeyVersion = bytes[mypos++];
         this.issuerFingerprint = bytes.subarray(mypos, bytes.length);
-        if (this.issuerKeyVersion === 5) {
+        if (this.issuerKeyVersion >= 5) {
```

Comment:
Shouldn't we also throw if we read a issuerKeyID packet from a non-v4 key? There is such a key in the sop tests (unhashed packets). We risk that the keyID set here is overwritten, otherwise.